chef_master/source/install_chef_automate.rst

```diff
@@ -361,13 +365,7 @@ new build node or runner. See the next section for installation instructions.
 
 .. note:: Your Chef Automate server will not be available for use until you either agree to apply the configuration, or manually run ``sudo automate-ctl reconfigure``.
 
-Finally, you must create an Enterprise on the Chef Automate server using the builder's SSH key generated by the ``automate-ctl setup`` command:
-
-   .. code-block:: bash
-
-      automate-ctl create-enterprise $ENTERPRISE_NAME --ssh-pub-key-file=/etc/delivery/builder_key.pub
-
-Copy the credentials somewhere safe. And in the ``$AUTOMATE\_SERVER``, if you don't have DNS, define it in ``/etc/hosts``:
+After setup successfullyl completes, copy the ``admin`` credentials and ``builder`` password to somewhere safe. And if you don't have DNS, define it in ``/etc/hosts``:
```

Comment:
Typo in successfully.

Side note: one thing that we usually do during setup is to run the setup command and output STDOUT to a file so it's saved. eg:

`automate-ctl setup .... 2>&1 | tee /etc/delivery/admin_credentials.txt`